### DISCUSSION TITLE
ARDUINO IDE Shows this library but no install button

### DISCUSSION THREAD

**Discussion by weledlight** [earlephilhower/arduino-pico#1224] [General]
Hi 
Wondered if anyone has encountered this issue?

**Comment by Papirosnik** [comment]
I believe the install button is there. In my case it's visible partially :)
I'm able to click on the version selector and press the install button as well (because I do know what is there of course)
I suppose you may try to make system font smaler or something like that...

<img width="289" alt="image" src="https://user-images.githubusercontent.com/1651549/221377422-ccaeba58-81bd-42da-b9e4-e502157f1585.png">

**Reply by fungizmos** [reply]
You should be able to drag the window splitter, to make that column wider.  The test will reflow and you can see the button.

**Comment by weledlight** [comment]
Appreciate your response, in the short term I downshifted to earlier IDE to install and then upshifted back as the library was in place. Thank you again for your help

Cheers

Mark

 

 

From: Slava K ***@***.***>
Reply to: earlephilhower/arduino-pico ***@***.***>
Date: Sunday, 26 February 2023 at 6:08 am
To: earlephilhower/arduino-pico ***@***.***>
Cc: MARK RILEY ***@***.***>, Author ***@***.***>
Subject: Re: [earlephilhower/arduino-pico] ARDUINO IDE Shows this library but no install button (Discussion #1224)

 

I believe the install button is there. In my case it's visible partially :)
I'm able to click on the version selector and press the install button as well (because I do know what is there of course)
I suppose you may try to make system font smaler or something like that...
